mixpanel.js

```diff
@@ -2339,15 +2339,15 @@ Globals should be all caps
     };
 
     /**
-     * Track an event.  This is the most important Mixpanel function and
-     * the one you will be using the most.
+     * Track an event.  This is the most important and
```

Comment:
You're right, I thought I saw single-space elsewhere. If you're feeling helpful it would be nice to get rid of them throughout, but don't feel obliged.